### ISSUE TITLE
Use Filter Daterange

### ISSUE THREAD

**Issue by fabriciofs** [zollero/el-search-table-pagination#33]
Hi, would you have any examples to use the filter with daterange, or the use filter-method?

Thanks

**Comment by zollero** [comment]
You wanna use `filter` to format the value of daterange?

**Comment by fabriciofs** [comment]
No, in Form Item Attributes I created a field with the itemType property filled with daterange. However in the datatable the data is not filtered. No matter what content I put, if I fill in, it always does not return any data.


`
forms: [
          { prop: 'date',
            label: 'Data',
            itemType: 'daterange'
          },
          { prop: 'status',
            label: 'Status',
            itemType: 'select',
            options: [
              { value: '', label: 'Todos' },
              { value: 'Disponível', label: 'Disponível' },
              { value: 'Entrega', label: 'Entrega' },
              { value: 'Transferida', label: 'Transferida' },
              { value: 'Finalizada', label: 'Finalizada' },
              { value: 'Bloqueada', label: 'Bloqueada' }
            ]
          }
        ]
`
